### PR TITLE
Updated list and find sub commands in user guide

### PR DESCRIPTION
**Pull request by shaowi**
Standardised list and find command on common attributes

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -666,44 +664,47 @@ Examples:
...
 > `find {keywords} [/byTag]`
 
-Find all modules/lectures/videos based on context whose tag list contains any tag that starts with any of the keyword(s)
+Find all modules whose tag list contains any tag that starts with any of the keyword(s)
 
 **Assumption:**\
-Module `CS2040S` has tags `["heavy", 'math']`\
-Lecture `Week 1` of `CS2040S` has tags `["Arrays", "Sorting"]`\
-Video `Vid 1` of `Week 1` of `CS2040S` has tags `["content"]`
+Module `CS2040S` has tags `["heavy", 'math']`
```
Comment (hingen) [inline docs/UserGuide.md:699]: A bit weirdly formatted, with how "Assumption" is bold. Possible to just combine with examples if you feel is necessary to state this?

File: docs/UserGuide.md [hunk 2]
```diff
@@ -713,69 +714,57 @@ Find all lectures in a specified module whose name starts with any of the keywor
...
 Examples:
 
-- `find week 1, week 2 /mod CS2040S` searches for lectures `Week 1` or `Week 2` from the lecture list of module `CS2040S`.
+- `find week 1, week 2 /mod CS2040S`
 
-### Find Lectures in a Module By Tag
+#### Find Lectures in a Module By Tag
 
-> `find {keywords} [/byTag /mod {module_code}]`
+> `find {keywords} /mod {module_code} [/byTag]`
```
Comment (hingen) [inline docs/UserGuide.md:721]: Is `/byTag` optional in this case?

File: docs/UserGuide.md (hunk 1 above)
Comment (shaowi) [reply docs/UserGuide.md:699]: i'll move it into the examples 👍

File: docs/UserGuide.md (hunk 2 above)
Comment (shaowi) [reply docs/UserGuide.md:721]: Yes its a typo, will correct it

**Review by jedidiahC — APPROVED** [review]
lgtm

**Pull request merged**